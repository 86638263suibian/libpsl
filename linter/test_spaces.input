// test:
// - leading space
// - trailing space, empty line with spaces
// - leading tab
// - trailing tab
// - line ends with CRLF
// - empty line with spaces

// ===BEGIN ICANN DOMAINS===

// example.com: https://www.iana.org/domains/reserved
 a.example.com
b.example.com 
	c.example.com
d.example.com	
e.example.com
  

// ===END ICANN DOMAINS===
