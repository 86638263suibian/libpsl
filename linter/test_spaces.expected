12: warning: Leading/Trailing whitespace: '// example.com: https://www.iana.org/domains/reserved'
13: warning: Leading/Trailing whitespace: ' a.example.com'
14: warning: Leading/Trailing whitespace: 'b.example.com '
15: warning: Leading/Trailing whitespace: '\tc.example.com'
16: warning: Leading/Trailing whitespace: 'd.example.com\t'
17: warning: Leading/Trailing whitespace: 'e.example.com^M'
19: warning: No PRIVATE section found
